### PR TITLE
Services 1778 Add collection transactions tests

### PR DESCRIPTION
**Pull request by danielailie**

### CODE REVIEW

**Review by andreibancioiu — DISMISSED** [review]

File: src/modules/nftCollections/tests/collections-transactions.service.spec.ts [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+
+    service = module.get<CollectionsTransactionsService>(CollectionsTransactionsService);
+  });
+
+  it('should be defined', () => {
+    expect(service).toBeDefined();
+  });
+
+  describe('issueToken', () => {
+    it('returns build transaction for issue non fungile with right arguments', async () => {
```
Comment (popenta) [inline src/modules/nftCollections/tests/collections-transactions.service.spec.ts:23]: typo in `fungile`. as discussed will be fixed in next pr

**Review by popenta — APPROVED** [review]

**Review by andreibancioiu — APPROVED** [review]

File: src/modules/nftCollections/tests/collections-transactions.service.spec.ts (hunk 1 above)
Comment (danielailie) [reply src/modules/nftCollections/tests/collections-transactions.service.spec.ts]: done

[the remark this answers is not in the document]
Comment (danielailie) [reply src/modules/nftCollections/tests/collections-transactions.service.spec.ts]: done

**Review by popenta — APPROVED** [review]
👍🏻

**Pull request merged**